bgpd/bgp_route.c

```diff
@@ -12594,6 +12599,12 @@ DEFUN(show_ip_bgp_afi_safi_statistics, show_ip_bgp_afi_safi_statistics_cmd,
 		json = json_object_new_object();
 		json_object_object_add(json, get_afi_safi_str(afi, safi, true),
 				       json_afi_safi);
+		json_object_int_add(json, "bgpBestPathCalls",
```

Comment:
Shouldn't be this in `bgp_table_stats_single()`? Because what if we have `show bgp vrf all statistics json`? Also, is this only for JSON output?

Reply:
No because it's not per afi/safi, it's per `struct bgp`.  I intentionally only made this as a json output because most operators will not care about this.

Reply:
Not afi/safi, but per-instance (BGP, let's say lots of VRFs), right?

Reply:
yes it's per `router bgp XXX` in the cli

Reply:
So this means that `show bgp vrf all statistics json` won't work as expected, correct?